views/templates/block/checklist.html.twig

```diff
@@ -202,7 +202,7 @@
                         {% endfor %}
                       </ul>
                     </td>
-                    <td>{{ icons.nok }}</td>
+                    <td>{{ icons.nok(psBaseUri) }}</td>
```

Comment:
I don't really like that we have to send the variable `psBaseUri` to each call, is there any other way?

Reply:
I'm not very comfortable with this approach too :/ another solution is to use the _context variable, (_context.psBaseUri) but this is worse for me

Reply:
Too bad. Marking as solved.